webpack.common.js

```diff
@@ -107,64 +119,63 @@ module.exports = {
     ],
   },
   plugins: [
-    new CleanWebpackPlugin(),
```

Comment:
This is no longer needed?

Reply:
This is probably still needed... `_static` can change even more than it used to... it does have to do more work, but otherwise, you can end up with 10s of copies of `index.xxxx.js`, etc. Perhaps only running it in `build:production`?

Reply:
I removed this because custom.css en theme.css also live in this directory. With the font extraction this is indeed back on the table.

Reply:
@bollwyvl , maybe the font's could be extracted in a dedicated folder, so this can be cleaned?

Reply:
I added back the clean plugin (otherwise the css / js files keep piling up after each change, which each times creates a new file with changed hash), but ensured that it keeps the theme.css file in the static path